dev-tools/mage/config.go

```diff
@@ -97,7 +82,20 @@ func (c ConfigFileParams) Empty() bool {
 // host for the generated configs. Defaults to linux/amd64.
 func Config(types ConfigFileType, args ConfigFileParams, targetDir string) error {
 	if args.Empty() {
-		args = defaultConfigFileParams
+		args = ConfigFileParams{
```

Comment:
@kvch I had to move this here to prevent the OSSBeatDir and LibbeatDir calls from invoking findElasticBeatsDir before apm-server's magefile has an opportunity to override UseVendor.